proxy/http/HttpConfig.cc

```diff
@@ -1312,6 +1312,7 @@ HttpConfig::startup()
   HttpEstablishStaticConfigLongLong(c.oride.cache_guaranteed_max_lifetime, "proxy.config.http.cache.guaranteed_max_lifetime");
 
   HttpEstablishStaticConfigLongLong(c.oride.cache_max_stale_age, "proxy.config.http.cache.max_stale_age");
+
```

Comment:
I've often looked at line 1314 and thought it really needed a blank line after it.

Reply:
This should be a 1-line PR but I'll let it slide.